### ISSUE TITLE
Pip installs held up by plone.app.lockingbehavior in Python 3.8

### ISSUE THREAD

**Issue by merbst** [plone/plone.app.lockingbehavior#9]
On ArchLinux x64 pacman installed python 3.8, when I do:
```
pip install plone.app.lockingbehavior
```
It fails, despite the presence of the 'versiontools' package!

```
Collecting plone.app.lockingbehavior                                                              Downloading https://files.pythonhosted.org/packages/35/05/e1bb69b4196bf781d45ff79a8d6f5f341cd6730e8ba18036ebdba699127a/plone.app.lockingbehavior-1.0.6.tar.gz                                   Installing build dependencies ... done
  Getting requirements to build wheel ... error                                                   ERROR: Command errored out with exit status 1:
   command: /usr/bin/python /usr/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmp87ilhx07
       cwd: /tmp/pip-install-1122u8mk/plone.app.lockingbehavior                                   Complete output (34 lines):
  Traceback (most recent call last):                                                                File "/usr/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 257, in <module>                                                                                                  main()                                                                                        File "/usr/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 240, in main      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/usr/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py", line 91, in get_requires_for_build_wheel                                                                               return hook(config_settings)
    File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 146, in get_requires_for_build_wheel                                                        return self._get_build_requires(config_settings, requirements=['wheel'])                      File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 127, in _get_build_requires                                                                 self.run_setup()
    File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 236, in run_setup
      super(_BuildMetaLegacyBackend,                                                                File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 142, in run_setup                                                                           exec(compile(code, __file__, 'exec'), locals())                                               File "setup.py", line 9, in <module>                                                              setup(
    File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 144, in setup                                                                                 _install_setup_requires(attrs)                                                                File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 132, in _install_setup_requires                                                               dist = distutils.core.Distribution(dict(                                                      File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 447, in __init__                                                                                  _Distribution.__init__(self, {
    File "/usr/lib/python3.8/distutils/dist.py", line 292, in __init__                                self.finalize_options()
    File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 740, in finalize_options
      ep.load()(self)                                                                               File "/tmp/pip-build-env-pe6_4yf0/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 747, in _finalize_setup_keywords                                                                  ep.load()(self, ep.name, value)                                                               File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2443, in load             return self.resolve()
    File "/usr/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2449, in resolve          module = __import__(self.module_name, fromlist=['__name__'], level=0)                       ModuleNotFoundError: No module named 'versiontools'                                             ----------------------------------------                                                      ERROR: Command errored out with exit status 1: /usr/bin/python /usr/lib/python3.8/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /tmp/tmp87ilhx07 Check the logs for full command output.
```

I even tried this to ensure that the latest build environment is available: pip install --force-reinstall versiontools pip setuptools pep517

**Comment by jugmac00** [comment]
The underlying problem seems to be that there is an error while trying to build a wheel for your platform.

E.g. on Ubuntu 18.04 / Py3.8 I get no error as there seems to be a pre-built wheel.

I am not sure how to proceed, but unlike the `versiontools` package, which seems to be abandoned for 8 years, this package here got some love from @mauritsvanrees - maybe he knows how to help.

Maybe there is an easy fix, but in the long run it would be a good idea to try to either reactivate `versiontools` or get rid of it as a dependency. Also it would be nice to have tox / travis config for this project.

P.S.: The output is a bit hard to read. You could try to wrap this kind of long output in three backticks.

**Comment by mauritsvanrees** [comment]
The traceback is unreadable, so I cannot comment on that one.

But it seems fine on my Py 3.8 (from August) as well:

```
$ python3.8 -m venv p38
$ cd p38
$ . bin/activate
$ python --version
Python 3.8.0b3+
$ pip install plone.app.lockingbehavior
...
Building wheels for collected packages: plone.app.lockingbehavior, plone.alterego, BTrees, plone.batching
  Building wheel for plone.app.lockingbehavior (PEP 517) ... done
  Created wheel for plone.app.lockingbehavior: filename=plone.app.lockingbehavior-1.0.6-cp38-none-any.whl size=7775 sha256=ae79476a8d1d8d660488999eafe01d5adc1e90a9af9b10a3b80cc0523279ef54
  Stored in directory: /Users/maurits/Library/Caches/pip/wheels/e9/98/62/ff1ee2caa07a0bdad8b2b89e6786d6a3cc3efb122998c6e57a
...
```
